prestoadmin/topology.py

```diff
@@ -208,6 +220,11 @@ def set_env_from_conf():
 
     env.user = conf['username']
     env.port = conf['port']
+    env.java8_home = None
```

Comment:
This is redundant. env.java8_home is guaranteed to be set in either the try block or the except block.